src/custom/utils/trade.ts

```diff
@@ -192,7 +191,7 @@ export async function signAndPostOrder(params: PostOrderParams): Promise<AddUnse
   if (allowsOffchainSigning) {
     const signedOrderInfo = await signOrder(unsignedOrder, chainId, signer)
     signingScheme =
-      signedOrderInfo.signingScheme === EcdsaSigningScheme.ETHSIGN ? SigningScheme.ETHSIGN : SigningScheme.EIP1271
+      signedOrderInfo.signingScheme === EcdsaSigningScheme.ETHSIGN ? SigningScheme.ETHSIGN : SigningScheme.EIP712
```

Comment:
Was that a typo or the signing scheme was wrong?

Because https://eips.ethereum.org/EIPS/eip-1271 is != https://eips.ethereum.org/EIPS/eip-712

Reply:
It's ok, I made a mistake before.
```
enum EcdsaSigningScheme {
    EIP712 = "eip712",
    ETHSIGN = "ethsign"
}
```

It can be only "eip712" or "ethsign"